CSharp/DevVmPowershell/DevVmPsModules/Cmdlets/AddAgentByNameModule.cs

```diff
@@ -61,14 +61,8 @@ protected override void ProcessRecordCode()
 			//Validate Input arguments
 			ValidateInputArguments();
 
-			IConnectionHelper connectionHelper = new ConnectionHelper(
-				relativityInstanceName: RelativityInstanceName,
-				relativityAdminUserName: RelativityAdminUserName,
-				relativityAdminPassword: RelativityAdminPassword,
-				sqlAdminUserName: SqlAdminUserName,
-				sqlAdminPassword: SqlAdminPassword);
 			IRestHelper restHelper = new RestHelper();
-			IAgentHelper agentHelper = new AgentHelper(connectionHelper, restHelper, RelativityInstanceName, RelativityAdminUserName, RelativityAdminPassword);
+			IAgentHelper agentHelper = new AgentHelper(restHelper, RelativityInstanceName, RelativityAdminUserName, RelativityAdminPassword);

```

Comment:
Why are we not using the IConnectionHelper and passing the credentials directly to the AgentHelper constructor? I think Using IConnectionHelper is better approach for Dependency Injection.

- [ ] Use IConnectionHelper

Reply:
I forgot connection helper has those properties, I will change it